dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -350,6 +350,8 @@ public void SpecifiedPlacement()
                     var targetRect = GetTargetBounds();
 
                     // All positions are valid
+                    // The following might not always work, so repeat.
```

Comment:
>// The following might not always work, so repeat. [](start = 20, length = 50)

Really? what happens? was that the case before this change?

Reply:
I am not entirely sure what the root cause is, however when running those tests locally, I noticed that the rect (and the red border we use to show it) don't always show up. The problem is that this leaves the TeachingTip to believe it has more space and fails the test.

Reply:
Why wasn't this causing the test to fail before this change?

Reply:
The test was very unreliable on my machine before that change, only 20% of the time it actually ran through.